src/ORM/Marshaller.php

```diff
@@ -77,6 +78,11 @@ protected function _buildPropertyMap($options)
             $assoc = $this->_table->association($key);
             if ($assoc) {
                 $map[$assoc->property()] = ['association' => $assoc] + $nested + ['associated' => []];
+            } else {
```

Comment:
Is it breaking coding convention? IMHO this is more readable and no unessesery if-checks are done if first IF is fulfilled.

And to be clear: You meant removing else and add additional if, not changing the original if right?